frontend/benefit/applicant/src/components/applications/forms/application/step2/ApplicationFormStep2.tsx

```diff
@@ -564,15 +574,22 @@ const ApplicationFormStep2: React.FC<DynamicFormStepComponentProps> = ({
             </$GridCell>
 
             <$GridCell $colSpan={2}>
-              <NumberInput
+              <TextInput
                 id={fields.employee.monthlyPay.name}
                 name={fields.employee.monthlyPay.name}
                 label={fields.employee.monthlyPay.label}
                 onBlur={formik.handleBlur}
-                onChange={formik.handleChange}
+                onChange={(e) =>
+                  formik.setFieldValue(
+                    fields.employee.monthlyPay.name,
+                    stringFloatToFixed2(e.target.value)
+                  )
+                }
                 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                 // @ts-ignore TODO: remove this ts-ignore when HDS-1013 is resolved
```

Comment:
Does TS compiler now complain if you remove this comment?

Reply:
removed, at least not complaining locally.